### DISCUSSION TITLE
#qj-format: table not rendering

### DISCUSSION THREAD

**Discussion by tech-architecti** [Rooyca/query-json#4] [Q&A] [answered]
Hello,

Love this tool. Thank you. 

Is anybody else having issues with the format flag. 

#qj-format: table (or list, or img) do not work for me.

**Comment by Rooyca** [comment]
Can you give me a little more context to see if we can solve the problem?

What data are you using? Can you give a mockup of the data? Can you provite the full codeblock you are using?

**Reply by Rooyca** [reply, marked as answer]
One more question:
Are you using `#qj-query` in your code-block? Because `#qj-format` only works with queries.
